### DISCUSSION TITLE
useAtlasEngine with PowerShell write-host -ForegroundColor -BackgroundColor

### DISCUSSION THREAD

**Discussion by mkanet** [microsoft/terminal#14925] [Q&A] [answered]
I just enabled `useAtlasEngine`.  However, I noticed that it doesn't respect any PowerShell font colors.  It's monochrome text.  I couldn't find any information on this other than `useAtlasEngine` supports color fonts.

What's the trick to getting `useAtlasEngine` to use font colors specified in PowerShell?

**Comment by zadjii-msft** [comment]
Uh, it should absolutely support colors... Can you share a screenshot, and maybe your settings.json?

**Comment by DHowett** [comment, marked as answer]
If you are using the font "Source Code Pro", please update it. It has a bug where it tells the font rendering subsystem that it contains its own colors (like an emoji), and then _it actually doesn't._

This results in Terminal trusting the colors inside the font (gray) and treating every letter like an emoji.

**Reply by mkanet** [reply]
Man, you totally guessed why it wasn't working!  I just updated Source Code Pro, and now it's working great.  Thank you for the quick response.
